packages/atlas/src/components/_comments/CommentInput/CommentInput.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { useEffect, useRef, useState } from 'react'
+import mergeRefs from 'react-merge-refs'
+import useResizeObserver from 'use-resize-observer'
+
+import { Information } from '@/components/Information'
+import { Text } from '@/components/Text'
+import { Button } from '@/components/_buttons/Button'
+import { useMediaMatch } from '@/hooks/useMediaMatch'
+
+import { Border, ButtonsContainer, Container, Flex, StyledCommentRow, StyledTextArea } from './CommentInput.styles'
+
+import { CommentRowProps } from '../CommentRow'
+
+export type CommentInputProps = {
+  processing: boolean
+  onComment?: () => void
+  onCancel?: () => void
+} & CommentRowProps
```

Comment:
Do we need all the CommentRowProps? I think the highlight is not needed

Reply:
not 100% sure but I don't think it will matter too much if not